src/include/pgexporter.h

```diff
@@ -84,15 +85,16 @@ extern "C" {
 #define HUGEPAGE_TRY 1
 #define HUGEPAGE_ON  2
 
-#define MAX_QUERY_LENGTH 1024
+#define MAX_QUERY_LENGTH      1024
+#define MAX_COLLECTOR_LENGTH  1024
 
 #define LABEL_TYPE      0
 #define COUNTER_TYPE    1
 #define GAUGE_TYPE      2
 #define HISTOGRAM_TYPE  3
 
-#define SORT_NAME  0
-#define SORT_DATA0 1
+#define SORT_NAME 0
+#define SORT_DATA 1
```

Comment:
Lets keep the `SORT_DATA0` name - we may want to sort by multiple data columns in the future